.github/workflows/lint.yml

```diff
@@ -66,3 +66,13 @@ jobs:
         with:
           tool: taplo-cli
       - run: taplo fmt --check
+  outdated:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/rust
+      - uses: taiki-e/install-action@v2
```

Comment:
Nit: I am starting to feel just slightly uncomfortable about number of external GithubAction dependencies we  are including in our CI.